### ISSUE TITLE
Add maconda (additional) masses

### ISSUE THREAD

**Issue by stanstrup** [stanstrup/commonMZ#2]

**Comment by rsalek** [comment]
Would be very cool @RJMW, also maybe MetaboLights could help as well.

**Comment by RJMW** [comment]
Very cool indeed! @rsalek & @stanstrup would be good to discuss this in more detail. Skype at some point?

**Comment by stanstrup** [comment]
Yes skype would be good. When would suit you? I am quite open this week.

If we are start maybe we can find a way to organize this such that new things can find their way back into Maconda?

**Comment by rsalek** [comment]
done
